### ISSUE TITLE
The Next Journey: Which Book to Read Together?

### ISSUE THREAD

**Issue by emptymalei** [neuronstar/general-discussions#1]

**Comment by emptymalei** [comment]
## Neuroscience + Machine Learning

我觉得可以学习一下结合 neuroscience + machine learning 这样的书。如果讨论类型的话，大致有：

1. Application of machine learning in neuroscience
2. Application of neuroscience in machine learning：neuroscience inspired hardware/software and how can we go furthure.
3. Machine learning for neuroscientists：单纯的学习 machine learning，这是另一个读书会的主题，所以这里应该不考虑。

找到一篇论文 [Toward an Integration of Deep Learning and Neuroscience](http://journal.frontiersin.org/article/10.3389/fncom.2016.00094/full)
还有一本有意思的书：Emergent Neural Computational Architectures Based on Neuroscience


## Computational Neuroscience

如果单纯这个的话，似乎可以进一步学习一下 theoretical neuroscience, 例如

```
Theoretical Neuroscience 
Computational and Mathematical Modeling of Neural Systems
by Peter Dayan and LF Abbott
```
[.](http://libgen.me/view.php?id=79706)

**这本书里面也有关于 neuroscience 和 machine learning 的讨论。（G. Hinton 的短文 [Machine learning for neuroscience](https://neuralsystemsandcircuits.biomedcentral.com/articles/10.1186/2042-1001-1-12) 推荐的一本书，提到里面有非常有趣的关于大脑和 machine learning 的观点。）**

目录：

```
Contents
Preface
Part I:    Neural Encoding and Decoding

1   Neural encoding I: Firing rates and spike statistics 
2   Neural encoding II: Reverse correlation and visual receptive fields 
3   Neural decoding 
4   Information theory

Part II:  Neurons and Neural Circuits

5   Model neurons I: Neuroelectronics 
6   Model neurons II: Conductances and morphology 
7   Network models
Part III: Adaptation and Learning

8   Plasticity and learning 
9   Classical conditioning and reinforcement learing 
10 Representational learning

Mathematical appendix 
References
```

**Comment by lxm1117** [comment]
https://github.com/CamDavidsonPilon/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers  。 这本既有一些模型的内容也侧重相关的编程。而且有一些ipython notebook 的例子。 书目如下。
Prologue: Why we do it.

Chapter 1: Introduction to Bayesian Methods Introduction to the philosophy and practice of Bayesian methods and answering the question, "What is probabilistic programming?"

Chapter 2: A little more on PyMC We explore modeling Bayesian problems using Python's PyMC library through examples. How do we create Bayesian models?

Chapter 3: Opening the Black Box of MCMC We discuss how MCMC, Markov Chain Monte Carlo, operates and diagnostic tools.

Chapter 4: The Greatest Theorem Never Told We explore an incredibly useful, and dangerous, theorem: The Law of Large Numbers.

Chapter 5: Would you rather lose an arm or a leg? The introduction of loss functions and their (awesome) use in Bayesian methods.

Chapter 6: Getting our prior-ities straight Probably the most important chapter. We examine our prior choices and draw on expert opinions craft priors.

Chapter X1: Bayesian methods in Machine Learning and Model Validation We explore how to resolve the overfitting problem plus popular ML methods.

Chapter X2: More PyMC Hackery We explore the gritty details of PyMC.

**Comment by ErbB4** [comment]
我推荐这本 The Elements of Statistical Learning

https://statweb.stanford.edu/~tibs/ElemStatLearn/printings/ESLII_print10.pdf



这是目录


1. Introduction
2. Overview of Supervised Learning
3. Linear Methods for Regression
4. Linear Methods for Classification
5. Basis Expansions and Regularization
6. Kernel Smoothing Methods
7. Model Assessment and Selection
8. Model Inference and Averaging
9. Additive Models, Trees, and Related Methods
10. Boosting and Additive Trees
11. Neural Networks
12. Support Vector Machines and Flexible Discriminants
13. Prototype   Methods   and Nearest-Neighbors
14. Unsupervised Learning
15. Random Forests
16. Ensemble Learning
17. Undirected Graphical Model
18. High-Dimensional Problems

我是觉得很多感兴趣的点都涵盖了，全书刷下来肯定会有帮助

**Comment by emptymalei** [comment]
@lxm1117 这本书我也 star 了，一直想读来着。

@ErbB4 我也想学习 statistical learning.

突然感觉好缺时间。
